docker-compose.yml

```diff
@@ -1,12 +1,11 @@
-version: '3'
-
 services:
   server:
     build:
       context: ./server
       dockerfile: ./Dockerfile
       args:
         BUILD_DEVELOPMENT: ${BUILD_DEVELOPMENT}
+    command: bash -c "python -m debugpy --listen 0.0.0.0:5678 app.py"
```

Comment:
For the record, I needed to replace this with:

```
command: bash -c "python -m flask run"
    ports:
      - "3001:3001"
```

Debuggy causing issues on my own machine at least, but making it a naieve flask server solves my issues for now. We SHOULD also figure out why debuggy is causing port issues on

Reply:
known issue, think it was introduced by python dockerfile bump from python3.8 to 3.11, if revert python 3.11 to 3.8, and do docker compose up, then things are fine:

==========
FROM python:3.8

ARG BUILD_DEVELOPMENT="False"

ENV PYDEVD_DISABLE_FILE_VALIDATION 1
LABEL version="1.1.0"
LABEL description="Demo of a Medicare claims data sample app"
...

since bump python to 3.11 is for security compliance, we probably need to figure out a work around to make debugpy function under python 3.11 docker.